src/components/SecondaryPanes/Breakpoints.css

```diff
@@ -53,18 +53,26 @@ html .breakpoints-list .breakpoint.paused {
   border-color: var(--breakpoint-active-color-hover);
 }
 
-.breakpoints-list .breakpoint-checkbox {
-  margin-inline-start: 0;
-  vertical-align: -2px;
-}
-
 .breakpoints-list .breakpoint-label {
+  max-width: calc(100% - 30px);
   display: inline-block;
   padding-inline-start: 2px;
-  padding-bottom: 4px;
   cursor: default;
 }
 
+.breakpoint-label .breakpoint-checkbox {
+  margin-inline-start: 0;
+  vertical-align: text-bottom;
+}
+
+.breakpoint-label .location {
+  width: 100%;
+  display: inline-block;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  padding: 1px 0;
```

Comment:
Provide a little bit vertical padding so that some text, like "g", won't get cut because of `overflow: hidden`, see the screenshot,
<img width="481" alt="screen shot 2017-09-17 at 5 06 43 pm" src="https://user-images.githubusercontent.com/5627487/30519499-addb7ea8-9bca-11e7-99d4-9aa25224771b.png">

Reply:
Thank you for the comment, maybe there is a more clear way to do this... Could we instead use overflow-x: hidden? ie)
```
overflow-x: hidden;
white-space: nowrap;
text-overflow: ellipsis;
```

Reply:
@codehag went for `overflow-x`, didn't add `white-space: nowrap` because it wouldn't wrap with overflow set.
p.s still need `padding: 1px 0` here because the cut was made by not enough height + overflow hidden. The height is 14px which is a bit not enough for "g". We could specify a height above 15px to compensate the cut but the vertical alignment would be no good.